docs/src/management-commands.rst

```diff
@@ -76,6 +76,20 @@ Duplicate all currently existing pages to make it easier to create production-li
     This command inherits from :class:`~integreat_cms.core.management.debug_command.DebugCommand`, so it is only available in debug mode.
 
 
+
+``copy_pois_and_contacts``
```

Comment:
I don't think this is an ideal name anymore, since a. it's possible to not copy contacts and b. it's also possible to copy events. But I'm also indecisive what name is better :thinking:
Either `copy_pois_events_and_contacts` which is long, or just `copy_pois` because the rest is optional, or `copy_content` but that way it sounds like pages could also be copied? :thinking:

Reply:
I decided for `copy_pois_events_and_contacts` :)